package/ios/Sources/CandleLinkSheetWrapperView.swift

```diff
@@ -8,8 +8,8 @@ struct CandleLinkSheetWrapper: View {
 
   @ObservedObject var viewModel: CandleLinkViewModel
 
-  init(appUser: Candle.Models.AppUser) {
-    self.candleClient = CandleClient(appUser: appUser)
+  init(appUser: Candle.Models.AppUser, accessGroup: String? = nil) {
```

Comment:
```suggestion
  init(appUser: Candle.Models.AppUser, accessGroup: String?) {
```
#pp this is multiple levels deep, no reason to provide a default...